src/vos/tests/vts_pm.c

```diff
@@ -1509,6 +1696,7 @@ static const struct CMUnitTest punch_model_tests[] = {
 	{ "VOS809: Small SGL test", small_sgl, NULL, NULL },
 	{ "VOS810: Conditionals test", cond_test, NULL, NULL },
 	{ "VOS811: Test vos_obj_array_remove", remove_test, NULL, NULL },
+	{ "VOS812: Minor epoch punch", minor_epoch_punch, NULL, NULL },
```

Comment:
I would suggest to split the new tests into three parts: EV type, SV type and rebuild simulation.